fetchai/ledger/api/bootstrap.py

```diff
@@ -0,0 +1,35 @@
+import requests
+
+
+def server_from_name(server_name):
+    # Get list of active servers
+    servers_response = requests.get('https://bootstrap.fetch.ai/networks/')
+    assert servers_response.status_code == 200, 'Failed to get network status'
+
+    # Check requested server is on list
+    available_servers = [s['name'] for s in servers_response.json()]
+    assert server_name in available_servers, 'Requested server not present on network: {}'.format(server_name)
```

Comment:
I think that I would be tempted to replace these asserts with some bespoke exceptions so that they can be handled from the users code

Reply:
Committed some more specific exceptions - let me know if you'd like these broken down further?